engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/runtime/batch/SetVariablesAsyncDto.ftl

```diff
@@ -0,0 +1,27 @@
+<@lib.dto>
+
+    <@lib.property
+        name = "processInstanceIds"
+        type = "array"
+        itemType = "string"
+        desc = "A list of process instance ids that define a group of process instances
+                to which the operation will set variables." />
+
+    <@lib.property
+        name = "processInstanceQuery"
+        type = "ref"
+        dto = "ProcessInstanceQueryDto" />
```

Comment:
Should we include the `desc` from https://github.com/camunda/camunda-docs-manual/pull/664 as well?